README.md

```diff
@@ -20,11 +20,12 @@ With the Code4z extension pack you can:
 - Access z/OS Datasets and z/OS Unix file systems, and submit JCLs.
 - Debug COBOL code for applications running in a CICS region.
 - View and download job output.
-- Issue TSO commands.
+- Issue TSO and MVS commands.
 
 ## What’s New
+* 2021-05-25 - [Explorer for Endevor](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.explorer-for-endevor) version 0.12.1, see [Changelog](https://marketplace.visualstudio.com/items/broadcomMFD.explorer-for-endevor/changelog).
+* 2021-05-24 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) version 1.15.1, see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog).
 * 2021-04-08 - [Debugger for Mainframe](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.debugger-for-mainframe) version 1.5, see [Changelog](https://marketplace.visualstudio.com/items/broadcomMFD.debugger-for-mainframe/changelog).
-* 2021-03-26 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) version 1.13.1, see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog).
 * 2021-03-15 - [COBOL Langage Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) version 0.18.0, see [Changelog](https://marketplace.visualstudio.com/items/broadcomMFD.cobol-language-support/changelog).
 

```

Comment:
as agreed, let's wait for the HLASM release, that will come early next week.

Reply:
@zimlu02, Dejan was suggesting that we add info on IDMS support to high-level descriptions of COBOL LS. What do you think of this? Would this PR be the right time to do it, or should we wait for the next COBOL release?

Reply:
nevermind - we will do this when we update Code4z next time after the COBOL release.